server/conf/i18n/messages.am

```diff
@@ -128,8 +128,6 @@ content.blockProgressFull={0} ከ{1} ተጠናቅቋል
 content.blockProgressLabel=የመተግበሪያ የሂደት ደረጃ
 # Heading on the application review page. The page shows all the answers the user inputted.
 heading.reviewAndSubmit=Review and submit
-# A message on a section indicating how many files have been uploaded in a section consisting of a file upload question
```

Comment:
It's fine for this PR, but going forward I'd advise making changes on messages files into its own PR to 1. make the PR smaller and easier to review 2. anything localization related is generally its own PR

Reply:
Will do